### DISCUSSION TITLE
I18n domComponents translations

### DISCUSSION THREAD

**Discussion by adam3278** [GrapesJS/grapesjs#5911] [Q&A] [answered]
Hi,
I want to translate my grapes DOM components via I18n module, but I have a small problem with it. When I do it by domComponents.names property in translation object, it overrides some components custom names (ex. countdown). How to achieve it on default (not grapes) dom components only? Thanks in advance.

**Comment by artf** [comment]
Can you please provide a reproducible demo?

**Reply by adam3278** [reply]
Here you are: https://codepen.io/adam3278/pen/Jjqbvbb

**Reply by artf** [reply, marked as answer]
Here is the [logic](https://github.com/GrapesJS/grapesjs/blob/b247a580751b626017c44ddac2981b9dff6772df/src/dom_components/model/Component.ts#L1522-L1542) behind it, as Countdown component doesn't have a default `name` defined it falls back to use the tagName one.

So, as a fix, you can extend the Countdown component (eg. in a plugin loaded after countdown one), or add the countdown in your i18n messages
```js
div: "Container",
countdown: "Countdown"
```
